src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,75 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readFile(fromFileName);
+        String report = createReport(dataFromFile);
+        writeReportToFile(report, toFileName);
+    }
+
+    private static String readFile(String fromFileName) {
+        StringBuilder stringBuilder = new StringBuilder();
+
+        try {
+            BufferedReader reader = new BufferedReader(new FileReader(fromFileName));
+            String value = reader.readLine();
+
+            while (value != null) {
+                stringBuilder.append(value).append(" ");
+                value = reader.readLine();
+            }
+        } catch (IOException e) {
+            throw new RuntimeException("Can't read data from file", e);
+        }
+
+        return stringBuilder.toString();
+    }
+
+    private String createReport(String dataFromFile) {
+        final String Supply = "supply";
+        final String Buy = "buy";
+        final String Result = "result";
+        String[] dataFromFileArray = dataFromFile.split(" ");
+        int supplyCount = 0;
+        int buyCount = 0;
+        int resultCount = 0;
+
+        for (int i = 0; i < dataFromFileArray.length; i++) {
+            String[] currentDataArr = dataFromFileArray[i].split(",");
+            String nameOfOperation = currentDataArr[0];
+            int currentCount = Integer.parseInt(currentDataArr[1]);
+
+            switch (nameOfOperation) {
+                case Supply:
+                    supplyCount += currentCount;
+                    break;
+                case Buy:
+                    buyCount += currentCount;
+                    break;
+                default:
```

Comment:
too primitive logic to use switch case, choose either if-else condition or ternary condition

Reply:
done